packages/google_sign_in/google_sign_in_platform_interface/lib/google_sign_in_platform_interface.dart

```diff
@@ -139,4 +139,22 @@ abstract class GoogleSignInPlatform extends PlatformInterface {
   Future<bool> requestScopes(List<String> scopes) async {
     throw UnimplementedError('requestScopes() has not been implmented.');
   }
+
+  /// Determines if the current user can access all [scopes].
+  ///
+  /// Optionally, an [accessToken] can be passed for applications where a
+  /// long-lived token may be cached (like the web).
+  Future<bool> canAccessScopes(
+    List<String> scopes, {
+    String? accessToken,
+  }) async {
+    throw UnimplementedError('canAccessScopes() has not been implmented.');
```

Comment:
Typo: implemented

Reply:
Oops, it's in a bunch of the methods. Could you mass-fix this file?

Reply:
> Oops, it's in a bunch of the methods. Could you mass-fix this file?

Absolutely, I copy-pasted and didn't notice the typo!